lib/modules/platform/bitbucket/index.ts

```diff
@@ -68,13 +70,18 @@ export async function initPlatform({
   }
   setBaseUrl(defaults.endpoint);
   renovateUserUuid = null;
+  const options: InternalHttpOptions = {
```

Comment:
The `options` object can also be instantiated without the `InternalHttpOptions` type, so I don't know which way is preferred.